.github/actions/rockylinux-build/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM rockylinux:8
+
+ENV IMAGEMAGICK_VERSION=7.1.0-45
```

Comment:
we probably want 7.1.0-16?
as per [docs](https://docs.alfresco.com/content-services/latest/support/)

Reply:
I'd stick to the 7.1.0-16 for now, the one that was used previously. If a need for update comes then a separate task would be created anyways.

And I don't think there's a need to join these 2 things, it might muddle up test failures and confuse version update with incorrect rpm install.